source/standards/source-code.html.md.erb

```diff
@@ -66,6 +66,29 @@ Update the application's README to explain why the repository has been archived,
 
 ## Working with Git
 
+### Default branch name
+
+Your repository's default branch should be called `main`.
+
+To configure git to use `main` for new repositories, first ensure you are using git 2.28 or later, then run:
+
+    git config --global init.defaultBranch main
+
+To migrate existing repositories from `master` to `main`, you can use
+[GitHub's branch renaming
+tool](https://github.com/github/renaming#renaming-existing-branches).
+
+Before you rename your default branch, you should consider the impact this will have on:
+
+ - internal users
+ - external users
+ - your CI system (for example, Concourse pipelines or GitHub actions)
```

Comment:
We should probably write out what CI stands for the first time we use it

Reply:
I will do this but CI feels like one of those acronyms where writing it out doesn't help much. CI no longer means "Continuous Integration", it means "running tests on a server"